### PR TITLE
Jetpack connect: already installed buttons

### PR DESCRIPTION
**Pull request by johnHackworth**
fixes https://github.com/Automattic/wp-calypso/issues/5225 (kind of)

Right now there are some fringe cases where we know that we are failing in detecting if the self-hosted site has jetpack installed or if it's installed and not active . Since we can't fix this (because we can't cover any possible configuration of the self-hosted site), we must make it easy for the user to indicate that they have / have not already installed jetpack if we are not detecting it properly.

This PR adds a couple of links to the instructions screens that allows to change between the activation and the install views:

image

image

And now I'm passing the wheel to @rickybanister  so he can add the final style/copy touches

Test live: https://calypso.live/?branch=add/jetpack-connect-already-installed-button

### CODE REVIEW

**Comment by rickybanister** [comment]
cc @richardmuscat to confirm this is indeed something we want to do

**Comment by rickybanister** [comment]
Discussed already with @johnHackworth but for posterity, I suggested moving these links to be part of the instructional text like so:

screen shot 2016-06-23 at 1 06 58 pm

screen shot 2016-06-23 at 1 08 46 pm

**Comment by johnHackworth** [comment]
@rickybanister done!

image

image

**Comment by rickybanister** [comment]
Ship that! Looks great.

File: client/signup/jetpack-connect/index.jsx [hunk 1]
```diff
@@ -286,13 +306,29 @@ const JetpackConnectMain = React.createClass( {
 					</div>
 					<Button onClick={ this.installJetpack } primary>{ this.translate( 'Install Jetpack' ) }</Button>
 					<div className="jetpack-connect__navigation">
-						{ this.renderBackButton() }
+						<div>{ this.renderBackButton() }</div>
```
Comment (rickybanister) [inline client/signup/jetpack-connect/index.jsx]: Why do we need this div?

Comment (johnHackworth) [inline client/signup/jetpack-connect/index.jsx]: oh, we don't... it's a leftover from the previous commit, I'll remove it!

**Pull request merged**